### ISSUE TITLE
[Feat]: it would be nice to add a column showing the rooms the nodes is in

### ISSUE THREAD

**Issue by andrewm4894** [netdata/netdata-cloud#716]
### Problem

https://netdata-cloud.slack.com/archives/CS3PB0VJ7/p1675424658451519

![image](https://user-images.githubusercontent.com/2178292/216597384-118be4af-0234-4277-bccf-6b1af2707612.png)

### Description

.

### Importance

nice to have

### Value proposition

1.

**Comment by amalkov** [comment]
Agree, with the link to the room setting page like `https://app.netdata.cloud/spaces/netdata-cloud/settings/rooms/k8s-prd-stable#`
